torch/distributed/fsdp/api.py

```diff
@@ -46,12 +46,22 @@ class ShardingStrategy(Enum):
       :class:`DistributedDataParallel` API. For gradients, this strategy
       synchronizes them (via all-reduce) after the backward computation. The
       unsharded optimizer states are updated locally per rank.
+    - ``HYBRID_SHARD``: Apply ``FULL_SHARD`` within a node, and data parallelism across
+        nodes. This results in reduced communication volume as expensive all-gathers and
+        reduce-scatters are only done within a node, and can be more performant for medium
+        sized models.
+    - ``HYBRID_SHARD_ZERO2``: Apply ``SHARD_GRAD_OP`` within a node, and data parallelism across
```

Comment:
Same comment as above:
```suggestion
    - ``HYBRID_SHARD_ZERO2``: Apply ``SHARD_GRAD_OP`` within a node and replicate parameters across
```

Reply:
[Easy] I recommend changing before landing